nginx/kitspace.template

```diff
@@ -59,6 +61,10 @@ server {
         }
         proxy_pass http://gitea:3000;
     }
+
+    location ~ / {
+        return 404;
+    }
```

Comment:
This seems to be taking precedence over /robots.txt. Maybe just remove the robots.txt location in the gitea server? If most things are 404 anyway then it's probably not needed.

Reply:
Resolved in 29820971.